Loenn/entities/power_line.lua

```diff
@@ -0,0 +1,34 @@
+local powerLine = {}
```

Comment:
These only render in-game if the nodes are placed at right angles, so movement should probably be limited somehow.

Also I asked about line visibility in the Loenn thread. I think these should either have the node lines always visible or just have some kind of connector drawn between them. Loenn has some line-drawing functions already (e.g. gondola nodes)

Reply:
Added a preview rendering of the path that should be shown in game